tasks/fix-cat2.yml

```diff
@@ -15,13 +15,31 @@
       - RHEL-07-010040
 
 - name: "MEDIUM | RHEL-07-010050 | PATCH | The operating system must display the Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a command line user logon."
-  command: "true"
+  copy:
+    src: "rhel_07_010050_issue_text"
+    dest: "/etc/issue"
+    owner: root
+    group: root
+    mode: 644
   tags:
       - cat2
       - medium
       - patch
       - RHEL-07-010050
 
+- lineinfile:
```

Comment:
Please Add:
 - name: "MEDIUM | RHEL-07-010050 | PATCH | The operating system must display the Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a command line user logon."

I know it's redundant, but that's the way we've decided to go. Want to make sure any user knows specifically what is being accomplished as the task is run (i.e. if they were to use step-mode, they can track)